### ISSUE TITLE
Shasum fail when running the curl install via bash

### ISSUE THREAD

**Issue by marcofranssen** [helm/chartmuseum#651]
```shell
$ curl https://raw.githubusercontent.com/helm/chartmuseum/main/scripts/get-chartmuseum | bash
Downloading https://get.helm.sh/chartmuseum--linux-amd64.tar.gz
Verifying checksum... SHA sum of /tmp/chartmuseum-installer-6ohEdG/chartmuseum--linux-amd64.tar.gz does not match. Aborting.
```

**Comment by marcofranssen** [comment]
Correct. So must be issue on Linux. Anyhow for Github actions I made now a setup action which simplifies adding chartmuseum to any workflow.

https://github.com/marcofranssen/setup-chartmuseum

**Comment by cbuto** [comment]
@marcofranssen Can you share what flavor of Linux you are using? I tested in an Ubuntu container and everything worked.

**Comment by marcofranssen** [comment]
It happened all of a sudden on a GitHub actions runner.

Using ubuntu-22.04

https://github.com/philips-labs/helm-charts/blob/5cbffdfd2172ef94c2ad053cd54087b46a6f84a3/.github/workflows/helm-chart-ci.yml#LL91C18-L91C26

In my current version of the workflow I'm using my github action that does not use the bash script anymore.
